packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -288,6 +295,22 @@ export class SingleAddressWallet implements ObservableWallet {
       tip$: this.tip$,
       transactionsHistoryStore: stores.transactions
     });
+
+    this.#resubmitSubscription = createTransactionReemitter({
+      confirmed$: this.transactions.outgoing.confirmed$,
+      rollback$: this.transactions.rollback$,
+      store: stores.volatileTransactions,
+      submitting$: this.transactions.outgoing.submitting$
+    })
+      .pipe(
+        mergeMap((transaction) => from(this.submitTx(transaction))),
+        catchError((err) => {
+          this.#logger.error('Failed to resubmit transaction', err);
+          return EMPTY;
```

Comment:
This is not ideal as SingleAddressWallet silently (end-users won't read the log) fails the transaction here.

We probably want to add some logic:
- resubmit on certain errors (probably with some timeout and maybe considering online status as well): `ProviderFailure.{Unknown,Unhealthy,ConnectionFailure}`
- emit something on any other errors, so that user could inform the user about failed transaction

We might want to do this in a separate ticket, we haven't considered this when estimating. What do you think @rhyslbw ?